addons/mail/static/src/core/web/activity_model.js

```diff
@@ -76,6 +76,7 @@ export class Activity extends Record {
         return activity;
     }
 
+    thread = Record.one("Thread");
```

Comment:
(minor) alphabetical order

Reply:
Is that field even used at all?

Reply:
Rather than sending thread on activity, it would be possible for thread data to return command to add activities on the inverse?

Reply:
It's the inverse of `this.activities = Record.many("Activity");` in thread. I think it's needed to insert the data directly in a relational field and if we don't provide it, it's not possible to use relational field for `thread` and `activity` anymore.

Reply:
If neither fields have inverse defined on them, they will not be sync together.